packages/ui-test-utils/src/utils/generateComponentExamples.js

```diff
@@ -22,8 +22,9 @@
  * SOFTWARE.
  */
 
-const { uid } = require('@instructure/uid')
-const generatePropCombinations = require('./generatePropCombinations')
+import { nanoid } from 'nanoid'
```

Comment:
Had to use a 3rd party lib for generating the keys, because the `generateComponentExamples` utility was moved from the `__examples__` package, to the `ui-test-utils` which is a dependency for the `@instructure/uid` package, and that would cause a dependency cycle.